prow/monitoring/mixins/grafana_dashboards/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+load("@io_bazel_rules_jsonnet//jsonnet:jsonnet.bzl", "jsonnet_library", "jsonnet_to_json")
+
+jsonnet_to_json(
+    name = "tide",
+    src = "tide.jsonnet",
```

Comment:
needs deps, see https://github.com/bazelbuild/rules_jsonnet

Reply:
The dependency is via `imports` (`-J`):

https://github.com/kubernetes/test-infra/blob/4e8e9dc4182b62b1afe638fc0be7ab86cdf890d4/prow/monitoring/mixins/grafana_dashboards/BUILD.bazel#L5-L7

In bash, the command is (`make` works fine)
https://github.com/kubernetes/test-infra/blob/4e8e9dc4182b62b1afe638fc0be7ab86cdf890d4/prow/monitoring/mixins/Makefile#L19-L21

My understand is that `dep` is for the `libsonnet` files in the same project while `imports` is for `vendor`. It would be nice to see an example of `rules_jsonnet` that works with the folder structure created by `jsonnet-bundler`.